task_sdk/tests/execution_time/test_task_runner.py

```diff
@@ -63,10 +87,15 @@ def test_recv_StartupDetails(self):
         assert decoder.request_socket.fileno() == w2.fileno()
 
 
-def test_parse(test_dags_dir: Path):
+def test_parse():
```

Comment:
That said, I don't think this test should be mocked -- we need something that is still exercising the `parse` code path.

Reply:
Sure, will not mock this one then.